lib/validatorCompiler.js

```diff
@@ -0,0 +1,33 @@
+const Ajv = require('ajv')
+
+const validatorCompiler = () => {
+  const ajv = new Ajv({
+    removeAdditional: true, // remove additional properties
+    useDefaults: true, // replace missing properties and items with the values from corresponding default keyword
+    coerceTypes: true, // change data type of data to match type keyword
+    nullable: true // support keyword "nullable" from Open API 3 specification.
+  })
+
+  ajv.addFormat('binary', {
+    type: 'string',
```

Comment:
Is string representation correct?

Reply:
can you pleaser explain more

Reply:
are you talking about comments

Reply:
It seems that a `binary` format would be represent by a binary type in JavaScript, but maybe I do not understand this correctly, could you please add a test?

Reply:
@mcollina have a look on The formats defined by the OAS
https://swagger.io/specification/